barretenberg/cpp/src/barretenberg/eccvm/eccvm_prover.hpp

```diff
@@ -62,7 +62,6 @@ class ECCVMProver {
     FF translation_batching_challenge_v; // to be rederived by the translator verifier
 
     SumcheckOutput<Flavor> sumcheck_output;
-    std::shared_ptr<CommitmentKey> commitment_key;
```

Comment:
these just seem to exist only to have shorter names, but the extra reference count makes it more annoying to free